airbyte-cdk/python/unit_tests/sources/file_based/conftest.py

```diff
@@ -1,35 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-import json
-import os
-import sys
-
-import pytest
-
-CONNECTION_SPECIFICATION = {
-    "type": "object",
-    "additionalProperties": True,
-}
-
-
-@pytest.fixture
-def json_spec():
```

Comment:
Originally the conftest.py was needed to define this extra test fixture for defining the spec.json for the test scenarios. However, this is no longer necessary since the spec should always be generated from the file-based source’s implementation and not a static file.